lib/rest-api.php

```diff
@@ -460,3 +460,36 @@ function gutenberg_silence_rest_errors() {
 	}
 
 }
+
+/**
+ * Include additional labels for registered post types
+ *
+ * @see https://core.trac.wordpress.org/ticket/45101
+ *
+ * @param array  $args      Arguments supplied to register_post_type().
+ * @param string $post_type Post type key.
+ * @return array Arguments supplied to register_post_type()
+ */
+function gutenberg_filter_post_type_labels( $args, $post_type ) {
+	$registered_labels = ( empty( $args['labels'] ) ) ? array() : $args['labels'];
+	if ( is_post_type_hierarchical( $post_type ) ) {
+		$labels = array(
+			'published_item'           => __( 'Page published.', 'gutenberg' ),
+			'published_item_privately' => __( 'Page published privately.', 'gutenberg' ),
+			'reverted_item_to_draft'   => __( 'Page reverted to draft.', 'gutenberg' ),
+			'scheduled_item'           => __( 'Page scheduled.', 'gutenberg' ),
+			'updated_item'             => __( 'Page updated.', 'gutenberg' ),
```

Comment:
@swissspidy @ocean90 Do you have any opinions about the naming of these label keys? (e.g. `published_item`)

Reply:
I'd probably name it `item_published` as the label itself.

Reply:
Agreed. This would be more in line with existing keys in `get_post_type_labels()` (`view_item`, `search_items`, `use_featured_image`, etc.).

SO I'd use `item_published`, `item_reverted_to_draft`, `item_scheduled`, etc.